README.md

```diff
@@ -45,12 +49,21 @@ way you might write a quick commit message. Your release notes should contain a
 clear overview of all changes that were added in this release.
 
 Tags don't push up to the remote server automatically. 
-To push your tag up, use `git push --tags`. 
+To push your tag up, use `git push --follow-tags`. 
 
 For more information on tags, check https://git-scm.com/book/en/v2/Git-Basics-Tagging.
 
 ## Getting Started
 
+### Using a Tagged Release
+
+When adding scaffold to a new project, make sure you use the latest tagged release 
+instead of just cloning master (as we did previously).
+
+To grab the lastest tagged release, navigate to the 'Releases' tab for this repo and download
```

Comment:
I think this one is okay to leave as 'tab', even though it technically isn't one. It indicates the general area to look for the link.